### ISSUE TITLE
Does this work with the new Gen4?

### ISSUE THREAD

**Issue by recoi1er** [nicholasrobinson/homebridge-smartcielo#21]
Before buying, was wanting to know if this would work the Gen4 from MrCool? This is the only plugin available, well there is another but hasn't been updated in 2 years!

**Comment by bverenkoff** [comment]
It doesn't appear this app has been touched in two years, and I can't get it to work, others on Reddit report similarly.

**Comment by recoi1er** [comment]
Ok I figured as much but was hoping maybe a notification to the dev might bring their focus back on it!

**Comment by isaac-webb** [comment]
Hey all,
I've commented on another issue in this repo as well, but I installed a couple of these units in my house this summer and wanted HomeKit. I took both of @nicholasrobinson's packages, updated them as best I could, and republished them. [`node-mrcool`](https://github.com/isaac-webb/node-mrcool) is the re-implemented library, and [`homebridge-mrcool`](https://github.com/isaac-webb/homebridge-mrcool) is the homebridge plugin. Right now it works (as far as I can tell) on my single zone 3rd gen, but I can't guarantee it'll work on newer stuff. That being said, if you went ahead and bought the 4th gen, I'd appreciate if you install and let me know the result/issues!
